internal/monitor/monitor.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2023 The NATS Authors
```

Comment:
we'll drop this soon as we can uptake the next release of nats cli where this is from.